src/glmfit.jl

```diff
@@ -464,7 +464,7 @@ end
 
 function StatsBase.fit!(m::AbstractGLM,
                         y;
-                        wts=nothing,
+                        wts::Union{AbstractVector{<:Real}, Nothing}=nothing,
```

Comment:
Maybe this should be restricted to

```suggestion
                        wts::Union{AbstractVector{<:Integer}, Nothing}=nothing,
```

(see https://github.com/JuliaStats/StatsBase.jl/pull/250#discussion_r2649569917)?